index.html

```diff
@@ -0,0 +1,247 @@
+                        <img class="" src="images/project3.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h1 class="title">Tonic</h1>
+                        <div class="role">
+                            <div class="role-name">CANOPY</div>
+                            <div class="role-dot"></div>
+                            <div class="role-job">Back End Dev</div>
+                            <div class="role-dot"></div>
+                            <div class="role-year">2015</div>
+                        </div>
+                        <div class="project-details">
+                            <p>
+                                A daily selection of privately personalized
+                                reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <ul class="skills">
+                            <li class="skill">html</li>
+                            <li class="skill">css</li>
+                            <li class="skill">javascript</li>
+                        </ul>
+                        <div class="button-click">
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project4.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h1 class="title">Multi-Post Stories</h1>
+                        <div class="role">
+                            <div class="role-name">CANOPY</div>
+                            <div class="role-dot"></div>
+                            <div class="role-job">Back End Dev</div>
+                            <div class="role-dot"></div>
+                            <div class="role-year">2015</div>
+                        </div>
+                        <div class="project-details">
+                            <p>
+                                A daily selection of privately personalized
+                                reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <ul class="skills">
+                            <li class="skill set1">html</li>
+                            <li class="skill set2">css</li>
+                            <li class="skill set3">javascript</li>
+                        </ul>
+                        <div class="button-click">
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section id="about" class="about">
+            <div class="left">
```

Comment:
- [ ] The content in this about section should be aligned with flexbox. Please fix this